src/javascript/app_2/App/Components/Routes/route_with_sub_routes.jsx

```diff
@@ -32,6 +32,7 @@ const RouteWithSubRoutes = route => {
             );
         }
 
+        document.title = `${localize(route.title)} | Binary.com`;
```

Comment:
do we want to use a placeholder or directly use `Binary.com`?

Reply:
@negarn 
it's possible to use placeholders but I don't exactly whether we need it or not?
@binary-hamedanchi @ashkanx What's your idea?

Reply:
We can just move it to a constant.

@morteza-binary Do we need to handle removing ` | ` separator in case of `route.title` is unavailable?